modules/configuring-the-training-operator-permissions-without-kueue.adoc

```diff
@@ -0,0 +1,157 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-training-operator-permissions-without-kueue_{context}"]
+= Configuring the Training Operator permissions without Kueue
+
+[role='_abstract']
+If you want to run a PyTorch training job in an environment where Kueue is not installed, you must provide the necessary access permissions to a service account.
+
+If you use Kueue to run PyTorch jobs, you can skip this section. 
+
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+ifndef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-on-data-science-projects_nb-server#creating-a-data-science-project_nb-server[Creating a data science project].
+endif::[]
+ifdef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#_using_data_science_projects[Creating a data science project].
+endif::[]
+
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+* You have access to a model.
+* You have access to data to train the model.
+
+.Procedure
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<admin_username>__ -p __<password>__
+----
+
+. Define a `ClusterRole` object in a YAML file called `pytorchjob_role.yaml` with the following contents:
```

Comment:
- change "called" to "named"
- does this mean that they need to create the `pytorchjob_role.yaml` file as part of this step?
If yes, rephrase to something like:
To configure access permissions for a new cluster role, create a `pytorchjob-role` object:
a. Create a YAML file named `pytorchjob_role.yaml`
b. Add the following `ClusterRole` object definition: